cachi2/core/package_managers/yarn/project.py

```diff
@@ -4,10 +4,21 @@
 It also provides basic utility functions. The main logic to resolve and prefetch the dependencies
 should be implemented in other modules.
 """
+import json
+import logging
```

Comment:
importing logging and initializing the logger needs to be done in the next commit where you actually make use of the logging interface.